SUPPORTED.md

```diff
@@ -0,0 +1,86 @@
+# SQL Supported Commands
+
+## Comparisson expressions
+- !=
+- ==
+- \>
+- <
+- \>=
+- <=
+- BETWEEN
+- IN
+- NOT IN
+- REGEXP
+
+## Null check expressions
+- IS NOT NULL
+- IS NULL
+
+## Grouping expressions
+- AVG
+- COUNT
+- MAX
+- MIN
+
+## Standard expressions
+- ALIAS (AS)
+- LITERAL
+- STAR (*)
+- CAST/CONVERT
+- CREATE TABLE
+- DESCRIBE/DESC/EXPLAIN
```

Comment:
I would specify the syntax we use for this (in our implementation `FORMAT=TREE` is required)